Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
```

Comment:
"Discounts" is plural, "Rabatt" is singular.  Which do you want?    Plural in German in "Rabatte"

Keep in mind that a "Discount Code" is not a "Discount" .   A Discount Code is usually called "Gutscheincode" or "Coupon"  (like in english, often called "Coupon" there, too).